var/spack/repos/builtin/packages/slate/package.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Slate(Package):
+    """The Software for Linear Algebra Targeting Exascale (SLATE) project is
+    to provide fundamental dense linear algebra capabilities to the US
+    Department of Energy and to the high-performance computing (HPC) community
+    at large. To this end, SLATE will provide basic dense matrix operations
+    (e.g., matrix multiplication, rank-k update, triangular solve), linear
+    systems solvers, least square solvers, singular value and eigenvalue
+    solvers."""
+
+    homepage = "https://icl.utk.edu/slate/"
+    hg      = "https://bitbucket.org/icl/slate"
+    maintainers = ['G-Ragghianti']
+
+    version('develop', hg=hg)
+
+    variant('cuda',   default=True, description='Build with CUDA support.')
+    # SLATE will eventually be buildable without intel-mkl by using other
+    #    providers of cblas and scalapack.
+    #variant('mkl',  default=True, description='Build using Intel MKL.')
+    variant('mpi',    default=True, description='Build with MPI support.')
+    variant('openmp', default=True, description='Build with OpenMP support.')
+
+    depends_on('cuda@9:', when='+cuda')
+    depends_on('intel-mkl')
+    # The cblas and scalapack dependencies are provided by MKL only for now.
+    #depends_on('cblas')
```

Comment:
Flake wants there to be a space between `#` and `depends_on` (all comments must be preceded with a space)